api.go

```diff
@@ -1,65 +1,48 @@
 package main
 
 import (
-	"encoding/json"
-	"fmt"
-	"io/ioutil"
-	"os"
-
-	"github.com/cs161-staff/project2-starter-code/client"
+    "encoding/json"
+    "fmt"
+    "io/ioutil"
+    "os"
+    "github.com/cs161-staff/project2-starter-code/client"
 )
 
 func main() {
-	// Read input commands from a json file
-	jsonFile, err := os.Open("input.json")
-	if err != nil {
-		fmt.Println(err)
-	}
-	fmt.Println("Successfully Opened json file")
-	defer jsonFile.Close()
-	byteValue, _ := ioutil.ReadAll(jsonFile)
-	var input map[string]string
-	json.Unmarshal([]byte(byteValue), &input)
-	command := input["command"]
-	username := input["username"]
-	password := input["password"]
+    // Read input commands from a json file
+    jsonFile, err := os.Open("input.json")
```

Comment:
should we take the location of the json file from the command line instead of hardcoding it? probably don't make much of a difference for our toy example